classes/template/PKPTemplateManager.inc.php

```diff
@@ -1662,16 +1662,24 @@ function getResourcesByContext($resources, $context) {
 	 */
 	function smartyPluckFiles($params, $smarty) {
 
+		// The variable to assign the result to.
+		if (empty($params['assign'])) {
+			error_log('Smarty: {pluck_files} function called without required `assign` param. Called in ' . __FILE__ . ':' . __LINE__);
+			return "";
```

Comment:
Missing the empty array assigning in smarty, and the return statement shouldn't return a string.